doc/app-METHOD.md

```diff
@@ -0,0 +1,80 @@
+<!---
+ Copyright (c) 2016 StrongLoop, IBM, and Express Contributors
+ License: MIT
+-->
+
+<h3 id='app.METHOD'>app.METHOD(path, callback [, callback ...])</h3>
+
+Routes an HTTP request, where METHOD is the HTTP method of the request, such as GET,
+PUT, POST, and so on, in lowercase. Thus, the actual methods are `app.get()`,
+`app.post()`, `app.put()`, and so on.  See below for the complete list.
+
+For more information, see the [routing guide](/guide/routing.html).
```

Comment:
Is it possible to make these links actually function in this repository? Very surly as soon as people start seeing the docs in this repository, there will be lots of simple pull requests, because people do it because they are looking for a quick way to get a commit into Express.js. Having broken links will surely create a bunch of PR churn.